apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState, useEffect} from 'react';
+import {useSelector} from 'react-redux';
+
+import {SimpleDropdown} from '@cdo/apps/componentLibrary/dropdown';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import i18n from '@cdo/locale';
+
+import styles from './teacher-navigation.module.scss';
+
+interface SectionsData {
+  [sectionId: number]: {
+    name: string;
+    hidden: boolean;
+  };
+}
+
+const TeacherNavigationBar: React.FunctionComponent = () => {
+  const sectionList = useSelector(
```

Comment:
```suggestion
  const sections = useSelector(
```
nit: mostly because it's not a list, it's an object.